consul/datadog_checks/consul/consul.py

```diff
@@ -213,7 +213,7 @@ def _check_for_leader_change(self, instance, instance_state):
 
                 self.event(
                     {
-                        "timestamp": int((datetime.now() - EPOCH).total_seconds()),
+                        "timestamp": int((datetime.utcnow() - EPOCH).total_seconds()),
```

Comment:
```suggestion
                        "timestamp": int(time.time()),
```

**Corollary**:

This like can be removed: `EPOCH = datetime(1970, 1, 1)` https://github.com/DataDog/integrations-core/pull/4173/files#diff-92c6a37ebfb4a295afacabd06258aff3R18

Reply:
Very true 👍